src/i18n/locales/zh-CN

```diff
@@ -0,0 +1,368 @@
+{
+  "about": {
+    "app_name": "Pocket Sync",
+    "new_firmware": "发现新的Pocket固件 v{version}",
+    "update_available": "软件有更新! {version}"
+  },
+  "app": {
+    "app_name": "Pocket Sync",
+    "connect_button": "连接到Pocket",
+    "not_pocket_tip": "目录无法识别, 请确认SD卡已经由Pocket完成初始化"
+  },
+  "clean_files": {
+    "close_button": "关闭",
+    "delete_files_button": "删除文件",
+    "files_found": "共找到{count, plural, =0 {0个可清理文件} one {# 个可清理文件} other {# 个可清理文件} }",
+    "title": "清理文件"
+  },
+  "core_info": {
+    "author": "作者",
+    "controls": {
+      "back": "返回列表",
+      "install": "安装",
+      "required_files": "必要文件",
+      "uninstall": "卸载",
+      "update": "更新"
+    },
+    "inputs": "按键提示",
+    "modal": {
+      "close": "关闭",
+      "input_core": "核心",
+      "no_settings_found": "未发现设置文件!",
+      "reset_all": "重置所有设置"
+    },
+    "requires_license": {
+      "all": "请注意!运行该核心需要取得许可授权",
+      "jotego": "JOTEGO betas需要JTBETA许可文件才能运行，可以通过JOTEGO的Patreon页面获取(https://www.patreon.com/jotego)。 \n 如果您取得了许可文件，请将jtbeta.zip放置在SD卡根目录，系统将自动根据必要文件的处理规则进行处理。"
+    },
+    "not_in_inventory": "核心{coreName}尚未加入清单",
+    "platform": {
+      "edit": "编辑",
+      "wikipedia": "维基百科"
+    },
+    "replaced": "此核心已经由<1>{replacementCore}</1>核心替代",
+    "platforms": "平台",
+    "please_wait": "请稍后，检查文件中...",
+    "release_date": "发布日期",
+    "releases": {
+      "release_history": "版本发布历史",
+      "show_previous": "显示之前的版本发布历史信息"
+    },
+    "required_files": "必要文件",
+    "required_files_count": "{count} / {total} 文件",
+    "settings": "配置",
+    "sponsor": "Sponsor",
+    "supports": "Supports",
```

Comment:
Looks like these two need a translation too, they turn up here:

<img width="367" alt="Screenshot 2023-10-13 at 15 48 19" src="https://github.com/neil-morrison44/pocket-sync/assets/2095051/b3a2c8d2-2b5c-4a36-b7b9-dd40a473c2ad">

and here

<img width="528" alt="Screenshot 2023-10-13 at 15 48 26" src="https://github.com/neil-morrison44/pocket-sync/assets/2095051/30d970df-f4cc-4a88-971b-37a1daed42b9">

Reply:
Done. Thx for the snapshot.